templates/mammo_form.html

```diff
@@ -0,0 +1,39 @@
+{% extends 'layout.html' %}
```

Comment:
I need some hand held explanation of how this form works because it just looks like magic to me. I will download it and then go through and update my comments if need be.